test/integration/puppetlabs/services/jruby_pool_manager/jruby_pool_int_test.clj

```diff
@@ -376,7 +300,7 @@
 
          (jruby-core/flush-pool! pool-context)
          ; wait until the flush is complete
-         (await (jruby-agents/get-pool-agent pool-context))
+         (is (jruby-testutils/wait-for-instances (jruby-internal/get-pool pool-context) 1))
```

Comment:
Did the `await` not work here anymore?  Seems like `wait-for-instances` could return true before the flush has started, depending upon timing.